README.md

```diff
@@ -1,2 +1,16 @@
 # credentials-io
 Library for reading/writing credentials
+
+## API
+
+### verify
```

Comment:
Check here for a more complete example: https://github.com/digitalbazaar/credentials-polyfill/blob/master/README.md#documentation